collection_json.py

```diff
@@ -270,7 +293,7 @@ def __getattr__(self, name):
 
     @property
     def properties(self):
-        """Return a list of names that can be looked up on the template."""
+        """Return a list of names that can be looked up on the item."""
```

Comment:
properties belongs to the item, looking it up in it's data field, I think it looks like it was copied over from the template properties accessor. The template object isn't involved for the Item class in this case. Can be confusing without see the context.